spec/controllers/idv/image_uploads_controller_spec.rb

```diff
@@ -129,9 +129,21 @@
           flow_path: 'standard',
         )
 
-        expect(@irs_attempts_api_tracker).not_to receive(:track_event).with(
+        expect(@irs_attempts_api_tracker).to receive(:track_event).with(
           :idv_document_upload_submitted,
-          any_args,
+          { address: nil,
+            date_of_birth: nil,
+            document_back_image_filename: nil,
+            document_expiration: nil,
+            document_front_image_filename: nil,
+            document_image_encryption_key: nil,
+            document_issued: nil,
+            document_number: nil,
+            document_state: nil,
+            failure_reason: { front: ['The selection was not a valid file.'] },
```

Comment:
Opinion question: I think the academically-correct thing to do here would be to re-use the `front: [I18n.t('doc_auth.errors.not_a_file')]` that's set up above, maybe moving it to a little `let(:error_msg)` assignment.

I have a slight preference for using the actual English string here, though, to better highlight the unusual (compared to other events) behavior where we are sending the actual translated string across. I think this makes it clearer.

Anyone feel strongly one way or the other?

Reply:
i think it's fine to hardcode the string like this, makes it easiest to change if something else changes